README.md

```diff
@@ -1,3 +1,79 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Usage
+
+Place a file named `.php_cs.dist` that has following content in your project's root directory.
+
+```php
+use Mollie\PhpCodingStandards\PhpCsFixer\Rules;
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->name('.php_cs.dist')       // Fix this file as well
+    ->in(__DIR__);
+
+return Config::create()
+    ->setFinder($finder)
+    ->setRiskyAllowed(true)
+    ->setRules(Rules::getForPhp71()); // use specific rules for your php version e.g.: getForPhp71, getForPhp72, getForPhp73
```

Comment:
can you place this comment above this line (in between the chain)? That way you avoid scrollbars in github's interface when actually reading the readme.